### PR TITLE
Updated content-container site examples to use appropriate header

### PR DESCRIPTION
**Pull request by msaikaushik1992**
### Summary
Updated content-container site examples to use appropriate header so that it is clear for the users.

### Additional Details
Related issue : https://github.com/cerner/terra-core/issues/677

### CODE REVIEW

File: packages/terra-site/src/examples/content-container/ContentContainerFill.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 import ContentContainer from 'terra-content-container';
-import Button from 'terra-button';
+import Heading from 'terra-heading';
 
 const display1 = <p key="1">display1display1display1display1display1display1display1display1display1display1display1display1display1display1</p>;
 const display2 = <p key="2">display2display2display2display2display2display2display2display2display2display2display2display2display2display2</p>;
```
Comment (StephenEsser) [inline packages/terra-site/src/examples/content-container/ContentContainerFill.jsx]: This doesn't necessarily need to be done on this pull request. But at some point we should update this display text to provide a better example. 

A quick update could be using lipsum text: http://www.lipsum.com/

SD / UX might be able to provide a better ideal template. Perhaps a header with a list or other more realistic header / content examples.

Comment (StephenEsser) [reply packages/terra-site/src/examples/content-container/ContentContainerFill.jsx]: Here are a couple examples I've thrown together with the ContentContainer. This view is composed of 3 ContentContainers.

image

Comment (msaikaushik1992) [reply packages/terra-site/src/examples/content-container/ContentContainerFill.jsx]: Updated display text using lipsum text for now. https://github.com/cerner/terra-core/commit/5e3d8f5b899dd44e291a630c354074c4c4c85a76

File: packages/terra-site/src/examples/content-container/ContentContainerFill.jsx [hunk 2]
```diff
@@ -1,24 +1,54 @@
...
-import Button from 'terra-button';
+import Heading from 'terra-heading';
 
-const display1 = <p key="1">display1display1display1display1display1display1display1display1display1display1display1display1display1display1</p>;
-const display2 = <p key="2">display2display2display2display2display2display2display2display2display2display2display2display2display2display2</p>;
-const display3 = <p key="3">display 3</p>;
-const display4 = <p key="4">display 4</p>;
-const display5 = <p key="5">display 5</p>;
-const display6 = <p key="6">display 6</p>;
+const display1 = (<p key="1">
```
Comment (emilyrohrbough) [inline packages/terra-site/src/examples/content-container/ContentContainerFill.jsx]: Nit: can we reduce the number of displays to 2 or 3. The example's pretty long for fake input that is repeated.

Comment (msaikaushik1992) [reply packages/terra-site/src/examples/content-container/ContentContainerFill.jsx]: Updated https://github.com/cerner/terra-core/commit/0f1aebd45306d036c18a279505d8d0aeef752712

Comment (emilyrohrbough) [reply packages/terra-site/src/examples/content-container/ContentContainerFill.jsx]: Thank you!

**Pull request merged**